config/routes.rb

```diff
@@ -37,8 +37,11 @@
   get 'people' => 'users#list'
   get 'users/role/:id' => 'users#list'
   match 'users/update' => 'users#update'
+  match 'users/:id/following' => 'users#following', as: :following
```

Comment:
@ebarry and @steviepubliclab, would you prefer "people" in the URL instead of "users"?